src/OpenTracing/Log/Fields.cs

```diff
@@ -0,0 +1,57 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="Fields.cs">
+//   Copyright 2017-2018 The OpenTracing Authors
+//   
+//   Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+//   in compliance with the License. You may obtain a copy of the License at
+//   
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+//   Unless required by applicable law or agreed to in writing, software distributed under the License
+//   is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+//   or implied. See the License for the specific language governing permissions and limitations under
+//   the License.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace OpenTracing.Log
+{
+    /// <summary>
+    ///     The following log fields are recommended for instrumentors who are trying to capture more
+    ///     information about a logged event. Tracers may expose additional features based on these
+    ///     standardized data points.
+    ///     <see href="https://github.com/opentracing/specification/blob/master/semantic_conventions.md" />
+    /// </summary>
+    public static class Fields
+    {
+        /// <summary>
+        ///     The type or "kind" of an error (only for event="error" logs). E.g., "Exception", "OSError"
+        /// </summary>
+        public static readonly string ErrorKind = "error.kind";
+
+        /// <summary>
+        ///     The actual Throwable/Exception/Error object instance itself. E.g., A java.lang.UnsupportedOperationException
+        ///     instance
+        /// </summary>
+        public static readonly string ErrorObject = "error.object";
+
+        /// <summary>
+        ///     A stable identifier for some notable moment in the lifetime of a Span. For instance, a mutex
+        ///     lock acquisition or release or the sorts of lifetime events in a browser page load described
+        ///     in the Performance.timing specification. E.g., from Zipkin, "cs", "sr", "ss", or "cr". Or,
+        ///     more generally, "initialized" or "timed out". For errors, "error"
+        /// </summary>
+        public static readonly string Event = "event";
+
+        /// <summary>
+        ///     A concise, human-readable, one-line message explaining the event. E.g., "Could not connect
+        ///     to backend", "Cache invalidation succeeded"
+        /// </summary>
+        public static readonly string Message = "message";
+
+        /// <summary>
+        ///     A stack trace in platform-conventional format; may or may not pertain to an error.
+        /// </summary>
```

Comment:
add the sample stack from the existing code

Reply:
Sorry, is this sample stack in the java code or the spec (`semantics.md`) or somewhere else?

Reply:
In the existing c# code - we had:
```csharp
        /// <summary>
        /// A stack trace in platform-conventional format; may or may not pertain to an error.
        /// <para />E.g.,
        /// <para />at App.Program.SomeMethod() in C:\app\Program.cs:line 14
        /// <para />at App.Program.Main(String[] args) in C:\app\Program.cs:line 9
        /// </summary>
        public const string Stack = "stack";
```

Reply:
resolved